backend/src/api-tests/reference/delete.test.js

```diff
@@ -0,0 +1,35 @@
+import { beforeAll, beforeEach, describe, expect, it } from '@jest/globals'
+import { login, logout, resetDatabase, resetDatabaseTimeout, send } from '../utils'
+
+describe('Deleting a locality works', () => {
+  beforeAll(async () => {
+    await resetDatabase()
+  }, resetDatabaseTimeout)
+  beforeEach(async () => {
+    await login()
+  })
+
```

Comment:
This is missing a `pool.end()`. See https://github.com/nowcommunity/nowdatabase/blob/5ef2c82582e32d8166d972b5753c3a4a36bd60e6/backend/src/api-tests/reference/create.test.ts#L6-L19